src/app/settings/reducer.ts

```diff
@@ -69,6 +69,9 @@ export interface Settings {
 
   /** Colorblind modes. */
   readonly colorA11y: string;
+
+  /** External source for wish lists. */
```

Comment:
Document that it's a URL. You may also want to validate the input to make sure it's really a URL.